samples/testing.fs

```diff
@@ -0,0 +1,45 @@
+module samples.unittest
+
+open Microsoft.Azure.WebJobs
+open DurableFunctions.FSharp
+open System
+
+[<FunctionName("SayHelloTest")>]
+let SayHello([<ActivityTrigger>] name) = 
+    sprintf "Hello %s!" name
+
+[<FunctionName("HelloSequenceTest")>]
+let RunWorkflow ([<OrchestrationTrigger>] context: DurableOrchestrationContextBase) = 
+    context |>    
+    orchestrator {
+      let! hello1 = Activity.callByName<string> "SayHello" "Tokyo"
+      let! hello2 = Activity.callByName<string> "SayHello" "Seattle"
+      let! hello3 = Activity.callByName<string> "SayHello" "London"
+
+      // returns ["Hello Tokyo!", "Hello Seattle!", "Hello London!"]
+      return [hello1; hello2; hello3]
+    }
+
+module WorkflowTests =
+
+  open Moq
+  open Expecto
+  
+  
+  [<Tests>]
+  let tests = 
+          testList "orchestration tests" [
+              testAsync "Orchestration test no records" {
+                   
+                  let mockContext = Mock<DurableOrchestrationContextBase>()
+                  mockContext.Setup(fun c -> c.CallActivityAsync<string>("SayHello","Tokyo")).ReturnsAsync("Hello Tokyo") |> ignore
+                  mockContext.Setup(fun c -> c.CallActivityAsync<string>("SayHello","Seattle")).ReturnsAsync("Hello Seattle") |> ignore
+                  mockContext.Setup(fun c -> c.CallActivityAsync<string>("SayHello","London")).ReturnsAsync("Hello London") |> ignore
```

Comment:
This looks quite mechanic... Can you think of any helper functions that we could provide to make it read smoother? Aren't there any F#-y ways to mock things?

Reply:
yeah I wanted to use the Foq library to mock as it has a much nicer  way of setting up the expectations using quotations but it doesn't work with the DurableOrchestrationContextBase - one of the properties has a sealed setter that causes Foq to fail (someone else already raised that issue so if it gets fixed I can swap it in)